python/pypipublish-test.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/zsh
```

Comment:
should this be inside the src dir?  Wouldn't it make more sense for this script to be either in the root of aiconfig or aiconfig/scripts

Reply:
I'll just remove this for now